added directory tests with files OneHandBandit_test.py, test_probabil…

OleksiiKH0240 opened the pull request:
…ities.txt; added probabilities.txt; modified Bandit.py, CustomExceptions.py, Pipfile, Pipfile.lock

 added getOneHandBanditReturn, setProbabilitiesFromTxtFile, genBaseProbabilities functions in class OneHandBandit in file Bandit.py;
 renamed genProbabilities function to genTurnProbabilities in class OneHandBandit in file Bandit.py;
 implemented unittests for some functions in class OneHandBandit in file Bandit.py;

OleksiiKH0240 commented:
не дуже розумію що мається на увазі під "рандомно генерувати ймовірності",
не розумію як їх генерувати, в залежності від коефіцієнту віддачі ігрового автомата,
не розумію, що означає (Монте-Карло) цилк гравця

mecchmatProjects reviewed (approved):
looks ok. Do You still have questions?

OleksiiKH0240 committed:
added folders user_interface, backend;

added folder user_interface with JavaScripts server:
added src\js folder for js files:
added Symbol.js for managing images;
added Reel.js for managing slots images and animation;
added Slot.js for control slots, spins, info fields, managing fields;
added index.js with const options dictionary, for linking object of Slot class with index.html;
added src\assets folder with all images that will used in site;
added src\css\style.css file with descriptions of look of some blocks in the document;
added folder backend:
added folder backend\doc\backend with documentation for python files from backend folder;
added test.py file for some test functions;
added folder test_files for test.py functions output;
added benchmark.py for function benchmark that measures working time of other functions;
added GenAlgorithm.py for genetic algorithm class;
added wsgi.py for flask server that will communicate with JavaScripts server;
added requirements.txt;
